File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLBoundaryFillableMapProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLBoundaryFillableMapProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> FillPoints;
        public String ImgURL;
        public String ImageWidth;
        public String ImageHeight;
        public String TabStopIndex;

        CCLBoundaryFillableMapProps()
        {
            FillPoints = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLBoundaryFillableMapProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLBoundaryFillableMapProps
    {
        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<List<int>> FillPoints { get; set; }
        public string ImgURL { get; set; }
        public int ImageWidth { get; set; }
        public int ImageHeight { get; set; }
        public int TabStopIndex { get; set; }

        public CCLBoundaryFillableMapProps()
        {
            FillPoints = new List<List<int>>();
        }
    }
}